storage/src/main/java/org/apache/uniffle/storage/handler/impl/HdfsClientReadHandler.java

```diff
@@ -127,21 +127,32 @@ public ShuffleDataResult readShuffleData() {
       init(fullShufflePath);
     }
 
-    if (readHandlerIndex >= readHandlers.size()) {
+    if (finished()) {
       return new ShuffleDataResult();
     }
 
-    HdfsShuffleReadHandler hdfsShuffleFileReader = readHandlers.get(readHandlerIndex);
-    ShuffleDataResult shuffleDataResult = hdfsShuffleFileReader.readShuffleData();
-
-    while (shuffleDataResult == null) {
-      ++readHandlerIndex;
+    HdfsShuffleReadHandler hdfsShuffleFileReader;
```

Comment:
HdfsShuffleReadHandler hdfsShuffleFileReader = null; ?

Reply:
I think no nessential, for unify?